spec/request/movie_request_spec.rb

```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+
+describe '#index', :type => :request do
+
+  context 'when movies available' do
+    before { get '/movies' }
+    it { expect(response).to have_http_status(:ok) }
+  end
+
+end
+
+describe '#create', :type => :request do
+  context 'when valid' do
+    
+    before {
+      post '/movies', :params => { :movie => { 

```

Comment:
[nitpick] Since Ruby 2.x we have nicer syntax with less symbols, as we're are lazy to type a lot :)
```suggestion
      post '/movies', params: { movie: { 
```
BTW, have you check the last changes with Rubocop linter?